internal/debug/flags.go

```diff
@@ -75,17 +77,6 @@ var (
 		Usage:    "Write logs to a file",
 		Category: flags.LoggingCategory,
 	}
-	backtraceAtFlag = &cli.StringFlag{
-		Name:     "log.backtrace",
-		Usage:    "Request a stack trace at a specific logging statement (e.g. \"block.go:271\")",
-		Value:    "",
-		Category: flags.LoggingCategory,
-	}
-	debugFlag = &cli.BoolFlag{
-		Name:     "log.debug",
-		Usage:    "Prepends log messages with call-site location (file and line number)",
-		Category: flags.LoggingCategory,
-	}
```

Comment:
Don't just remove them, they first should be deprecated: if a user uses them, we should print out "Hey log.backtraceat is deprecated, do not use this option or we will come over there and kick your ass". 

And then they get a release or fifteen to remove the flag from their config, before we actually drop it and refuse to start up due to `flag provided but not defined: -log.backtrace`